client/src/lib/components/DatasetForm/DatasetForm.svelte

```diff
@@ -250,83 +291,84 @@
-      id="contactEmails-legend"
-    >
-      E-mail(s) de contact
-      <span class="fr-hint-text" id="contactEmails-desc-hint">
-        Vous pouvez ajouter des adresses e-mail personnelles en complément
-        l'dresse fonctionnelle. Ces emails seront régulièrement vérifiés afin
-        d'assurer la bonne maintenabilité des jeux de données.
-      </span>
-    </legend>
+    <label class="fr-label" for="lastUpdatedAt">
+      Date de la dernière mise à jour
+      <RequiredMarker />
+    </label>
 
-    <ul class="fr-fieldset__content fr-raw-list contact-entries fr-mb-3w">
-      {#each $form.contactEmails as _, i}
-        <li role="presentation">
-          <label for="contactEmails-{i}" hidden>Contact {i}</label>
-          <input
-            class="fr-input"
-            class:fr-input--error={contactEmailErrors[i]}
-            aria-describedby={contactEmailErrors[i]
-              ? `contactEmails-${i}-desc-error`
-              : null}
-            type="email"
-            id="contactEmails-{i}"
-            name="contactEmails-{i}"
-            on:change={handleChange}
-            on:blur={handleChange}
-            bind:value={$form.contactEmails[i]}
-          />
-          <button
-            type="button"
-            class="fr-btn fr-btn--secondary fr-fi-delete-fill"
-            title="Supprimer cet e-mail de contact"
-            on:click|preventDefault={() => removeContactEmail(i)}
-          />
-          {#if contactEmailErrors[i]}
-            <p id="contactEmails-{i}-desc-error" class="fr-error-text">
-              {contactEmailErrors[i]}
-            </p>
-          {/if}
-        </li>
-      {/each}
-    </ul>
+    <div class="fr-input-wrap fr-fi-calendar-line">
+      <input
+        class="fr-input {$errors.lastUpdatedAt ? 'fr-input--error' : ''}"
+        aria-describedby={$errors.lastUpdatedAt
+          ? "entrypoint-service-desc-error"
+          : null}
+        type="date"
+        id="lastUpdatedAt"
+        name="lastUpdatedAt"
+        required
+        on:change={handleChange}
+        on:blur={handleChange}
+        bind:value={$form.lastUpdatedAt}
+      />
 
-    <button
-      type="button"
-      class="fr-btn fr-btn--secondary fr-fi-edit-fill fr-btn--icon-left contact-entries-add"
-      on:click|preventDefault={() => addContactEmail()}
+      {#if $errors.lastUpdatedAt}
+        <p id="title-desc-error" class="fr-error-text">
+          {$errors.lastUpdatedAt}
+        </p>
+      {/if}
+    </div>
+  </div>
+
+  <div class="fr-select-group">
+    <label class="fr-label" for="updateFrequency"
+      >Fréquence de mise à jour <RequiredMarker /></label
     >
-      Ajouter un contact
-    </button>
-  </fieldset>
+    <select
+      class="fr-select"
+      required
+      bind:value={$form.updateFrequency}
+      id="updateFrequency"
+      name="updateFrequency"
+      on:change={handleChange}
+      on:blur={handleChange}
+    >
+      <option>Selecectionner une option</option>
```

Comment:
Typo + il manque les attributs pour faire de cette option un véritable placeholder, cf : https://gouvfr.atlassian.net/wiki/spaces/DB/pages/223019306/Liste+d+roulante+-+Select#Liste-d%C3%A9roulante-avec-texte-d%E2%80%99aide

```suggestion
      <option value="" selected disabled hidden>Sélectionner une option</option>
```